pro/admin-tasks.md

```diff
@@ -0,0 +1,696 @@
+example, they hide a report by accident and wish to revert it to public, they must contact a team
+member with the appropriate permissions.
+
+#### Removing users’ names
+
+When users make a report on FixMyStreet, they may choose not to display their name on the
+public web page. If a user accidentally chooses to display their name and then wishes to remove it,
+they can do so themselves when logged in to FixMyStreet.
+
+However, if necessary, council staff can also remove a name from a public report on behalf of the
+user: see ‘[moderating reports](#deal-undesirable-content)’.
+
+#### Adding a user to the abuse list
+
+<span class="pro-task__permissions">Permissions required:​ ​User must be marked as staff, ‘Edit reports’ must be ticked.</span>
+
+You can add an abusive user's email to the abuse list, which automatically hides any reports they
+create, and means that their reports are not sent. Instead, the user sees a message that there was
+an error in confirming their report.
+
+Staff with the required permissions can ban a user directly from a report page, using the ‘Ban user’
+button.
+
+<img alt="Ban a user directly from a report page" src="/assets/img/pro-user-guide/ban-user-report-page.png" class="pro-screenshot" />
+
+#### Removing users from the banned list
+
+When a user has been banned, an ‘unban’ button will be visible on their records and reports. This
+restores them to the status of a standard user, but it does not have any effect on reports which
+were made during the period when the user was banned — these remain unsent.
+
+</div>
+
+<div class="pro-task" markdown="1" id="correct-reporter-errors">
+
+### Correcting reporter errors
+
+#### Changing a report’s category
+
+<span class="pro-task__permissions">Permissions required: User must be marked as staff, and ‘edit report category’ must be ticked.</span>
+
+If the user has chosen the wrong category for their report, when logged in, staff with the appropriate permissions can change it from the dropdown to the right of the report.
+
+<img alt="Change the category of a report" src="/assets/img/pro-user-guide/change-category-report.png" class="pro-screenshot" />
+
+If changing the category of the report results in it becoming the responsibility of another authority,
+the report will be re-sent to that authority. Do not mark it as closed, even though you have now
+dealt with it as far as your council is concerned; from the other authority's point of view, and for
+the person who made the report, the status is still open.
+
+However, if the report remains your council’s responsibility, the category change does not re-send
+the report unless the send method has changed (e.g. in the case of councils where some reportsare sent via the Open311 protocol and others by email).
+
+#### Changing the location
+Permissions required: User must be marked as staff, and ‘Edit report priority’ must be ticked.
```

Comment:
Missing `<span class="pro-task__permissions">`

Reply:
Also, I don't think priority permission is necessarily needed to do this? You need to be able to see the inspect form, so one or more of report_inspect, report_edit_category or report_edit_priority. I think.

Reply:
> Missing `<span class="pro-task__permissions">`

Updated